app/models/manageiq/providers/openstack/cloud_manager/vm.rb

```diff
@@ -87,6 +87,10 @@ def validate_migrate
     validate_supported
   end
 
+  def memory_mb_available?
+    true
+  end
```

Comment:
it's the condition in the yaml charts, otherwise the chart will not show up in the UI